Content.Client/StationEvents/IStationEventManager.cs

```diff
@@ -0,0 +1,13 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
Unnecessary imports.

```suggestion
using System.Collections.Generic;
```